python-checks/src/main/java/org/sonar/python/checks/PandasAddMergeParametersCheck.java

```diff
@@ -50,24 +88,39 @@ private static void verifyMergeCallParameters(SubscriptionContext ctx) {
     CallExpression callExpression = (CallExpression) ctx.syntaxNode();
     Optional.ofNullable(callExpression.calleeSymbol())
       .map(Symbol::fullyQualifiedName)
-      .filter(MERGE_METHODS::contains)
-      .ifPresent(fqn -> missingArguments(ctx, callExpression));
+      .filter(METHODS::contains)
+      .ifPresent(fqn -> missingArguments(fqn, ctx, callExpression));
   }
 
-  private static void missingArguments(SubscriptionContext ctx, CallExpression callExpression) {
+  private static void missingArguments(String fullyQualifiedName, SubscriptionContext ctx, CallExpression callExpression) {
     List<String> parameters = new ArrayList<>();
-    if (TreeUtils.argumentByKeyword("how", callExpression.arguments()) == null) {
-      parameters.add("how");
+    if (argumentIsMissing(fullyQualifiedName, Keywords.HOW, callExpression.arguments())) {
+      parameters.add(Keywords.HOW.getKeyword());
     }
-    if (TreeUtils.argumentByKeyword("on", callExpression.arguments()) == null) {
-      parameters.add("on");
+    if (argumentIsMissing(fullyQualifiedName, Keywords.ON, callExpression.arguments())) {
+      parameters.add(Keywords.ON.getKeyword());
     }
-    if (TreeUtils.argumentByKeyword("validate", callExpression.arguments()) == null) {
-      parameters.add("validate");
+    if (argumentIsMissing(fullyQualifiedName, Keywords.VALIDATE, callExpression.arguments())) {
+      parameters.add(Keywords.VALIDATE.getKeyword());
     }
     if (!parameters.isEmpty()) {
-      ctx.addIssue(callExpression, String.format(messages.get(parameters.size() - 1), parameters.toArray()));
+      parameters.add(fullyQualifiedName.substring(fullyQualifiedName.lastIndexOf('.') + 1));
+      ctx.addIssue(callExpression, String.format(MESSAGES.get(parameters.size() - 2), parameters.toArray()));
     }
+  }
+
+  private static boolean argumentIsMissing(String fullyQualfiedName, Keywords keyword, List<Argument> arguments) {
```

Comment:
Nitpick: no need to change anything. Usually, when naming a method returning a boolean I prefer starting with the verb so `is`, `has` etc... like `isArgumentMissing`.

Reply:
Fixed in b62ad8a311e6cf817d950df11b14d0b282135a79.